src/plugins/menu/_noscript.scss

```diff
@@ -10,22 +10,28 @@
 	display: none !important;
```

Comment:
To be replace by @extend %noscript-display-block-important

The %noscript-display-block-important is defined in the global [noscript at line 7](https://github.com/wet-boew/wet-boew/blob/master/src/base/noscript.scss#L7) and also used by other plugin.